products/PurePhone/CMakeLists.txt

```diff
@@ -102,6 +102,8 @@ add_hex_target(PurePhone)
 
 include(BinaryAssetsVersions.cmake)
 include(DownloadAsset)
+include(DownloadAsset2)
+download_asset2(${CMAKE_CURRENT_SOURCE_DIR}/assets.json ${PROJECT_SOURCE_DIR} ${MUDITA_CACHE_DIR})
```

Comment:
the same

Reply:
https://github.com/mudita/MuditaOS/pull/3210#discussion_r746581546